### PR TITLE
Mini design doc for node labels

### PR DESCRIPTION
**Pull request by taneyland**
*Issue #, if available:*

*Description of changes:*
This is the mini design doc for adding support to add labels to nodes

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: designs/node-labels.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Add support for node labels
+
+## Introduction
+
+**Problem:** Within an EKS-A cluster, the user might want to run specific workloads only on certain nodes.
+A great way to do this is by assigning labels to the desired nodes.
+However, there is currently no way for users to add labels to their nodes.
```
Comment (g-gaston) [inline designs/node-labels.md:7]: super nit: it's actually possible to add labels to nodes manually, post cluster creation. The problem right now is those labels are not persistent. If the node or machine are deleted or if the user upgrades the cluster (which rolls new machines hence new nodes) those labels will be lost and will need to be added again.

**Review by g-gaston — APPROVED** [review]
This is great! :shipit:

File: designs/node-labels.md (hunk 1 above)
Comment (taneyland) [reply designs/node-labels.md:7]: Thanks for pointing that out! I think that's pretty important to note so I'll update the doc to mention this

**Review by jaxesn — APPROVED** [review]
Looks good!

/lgtm

**Pull request merged**